File: numprimo.py
def é_primo(n):
    if n <= 1:
        return False
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

n = int(input("Qual número você quer verificar se é primo: "))
if é_primo(n):
    print(f"{n} é primo.")
else:
    print(f"{n} não é primo.")

File: numprimo.c
#include <stdio.h>
#include <stdbool.h>

bool é_primo(int n) {
    if (n <= 1) {
        return false;
    }
    for (int i = 2; i < n; i++) {
        if (n % i == 0) {
            return false;
        }
    }
    return true;
}

int main(void) {
    int n;
    printf("Qual número você quer verificar se é primo: ");
    scanf("%d", &n);
    if (é_primo(n)) {
        printf("%d é primo.\n", n);
    } else {
        printf("%d não é primo.\n", n);
    }
    return 0;
}
